CHANGELOG.md

```diff
@@ -80,7 +82,7 @@
 - `k8s_observer`: discover k8s.node endpoints (#6820)
 - `redisreceiver`: Add missing description fields to keyspace metrics (#6940)
 - `redisreceiver`: Set start timestamp uniformly for gauge and sum metrics (#6941)
-- `kafkaexporter`: Allow controlling Kafka acknowledgment behaviour  (#6301)
+- `kafkaexporter`: Allow controlling Kafka acknowledgment behaviour (#6301)
```

Comment:
All the changes below made by auto formatting? Maybe worth separating to another PR, but up to you.